src/app/delegates/components/DelegatorFilter.tsx

```diff
@@ -19,8 +17,10 @@ export default function DelegatorFilter() {
   const { setIsDelegatesFiltering } = useAgoraContext();
   const { address } = useAccount();
 
-  const filterParam = searchParams?.get(FILTER_PARAM) || DEFAULT_FILTER;
-  const delegateeFilterOptions: { value: string; sort: string }[] = [
+  const currentParam = searchParams?.get(FILTER_PARAM);
+  const filterParam = currentParam ? "my_delegates" : "all_delegates";
+
+  const delegateeFilterOptions = [
```

Comment:
Not in the scope of this PR: lets move these outside of components next time when we add so we don't break any memoisation of child components. Here in this case it is fine since the child component is anyway not memoized, but its better practice